src/main/scala/uk/gov/nationalarchives/tdr/api/graphql/fields/ConsignmentFields.scala

```diff
@@ -13,8 +13,11 @@ import uk.gov.nationalarchives.tdr.api.graphql.fields.FieldTypes._
 object ConsignmentFields {
   case class Consignment(consignmentid: Option[UUID] = None, userid: UUID, seriesid: UUID)
   case class AddConsignmentInput(seriesid: UUID)
+  case class ConsignmentFileMetadataProgress(processedFiles: Int, totalFiles: Int)
```

Comment:
Now I'm looking at this in the context of the other GraphQL queries, I'm wondering if we should actually integrate this into the Consignment endpoint rather than creating a new one. 🤔 

Do you want to pair on this again? There are some Sangria tricks we can use, and it's probably easier to go through them in person. It shouldn't affect the rest of your code - it's just about this GraphQL layer.

Reply:
I think that could be a really good idea yeah! :raised_hands: